Changelog.md

```diff
@@ -1,3 +1,13 @@
+Unreleased
+-------------------------
+* [Enhancement] Update requirements to keep in sync with `edx-platfom` 
+  and support the Open edX Redwood release.
+* [BREAKING CHANGE] Update the import statements for `xblock-utils`.
```

Comment:
Just checking: can't we make this a non-breaking change by using a `try`/`except` block on the import statement and catching the `ImportError`?